recipes/r-phewas/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = '0.12.1' %}
+{% set name = 'PHeWAS' %}
+{% set hash = '7063af0c3759e37e44f4ee2b92e22e579d9c58e15f0a529de593540489878544' %}
+
+package:
+  name: 'r-{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: 'v0.12.tar.gz'
+  url:
+    - 'https://github.com/{{ name }}/{{ name }}/archive/v0.12.tar.gz'
+  
+  sha256: '{{ hash }}'
+
+build:
+  skip: True    # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I think we have to change that to:

```
- {{ compiler('r_c') }}
```

Reply:
Hi @marcelotrevisani ,

From what I gather from https://bioconda.github.io/troubleshooting.html it looks something to do with the build environment and conda-build=3 new features. i will look to ammend the recipe to be compatible with the former version.

Thanks,